components/common/src/templating/package.rs

```diff
@@ -255,7 +255,7 @@ fn default_user_and_group() -> Result<(String, String)> {
                 _ => {
                     return Err(Error::PermissionFailed(
                         "Can't determine current user:group".to_string(),
-                    ))
+                    ));
```

Comment:
I thought you [were in favor of](url
https://github.com/habitat-sh/habitat/pull/6100/files#r252381020) 
https://rust-lang-nursery.github.io/rust-clippy/master/index.html#needless_return 😉 

Seriously though, this could be
```rust
            match (user, group) {
                (Some(user), Some(group)) => {
                    debug!("Running as {}/{}", user, group);
                    Ok((user, group))
                }
                _ => Err(Error::PermissionFailed(
                    "Can't determine current user:group".to_string(),
                )),
            }
```
right?